cryptography/src/sha256/mod.rs

```diff
@@ -58,6 +58,14 @@ impl Clone for Sha256 {
     }
 }
 
+impl Sha256 {
```

Comment:
Is it worth putting a #[cfg(test)] around this?

Reply:
I "think" that if you do that, you can't use it in other crates?

Reply:
For something similar in `Scheme`, I added a `# Warning`: https://github.com/commonwarexyz/monorepo/blob/40b0db0fc5aa7e725d03a6f510c6ea4f76f33328/cryptography/src/lib.rs#L67-L76